neptune/management/internal/api.py

```diff
@@ -558,6 +560,8 @@ def remove_project_service_account(
 
     backend_client = _get_backend_client(api_token=api_token)
     project_identifier = normalize_project_name(name=name, workspace=workspace)
+    if workspace is None:
```

Comment:
You can use `extract_project_and_workspace`

Reply:
Actually fixed it when rebasing.